metrics/web/lib/project_groups/presentation/widgets/project_group_card.dart

```diff
@@ -59,34 +59,54 @@ class _ProjectGroupCardState extends State<ProjectGroupCard> {
                     mainAxisAlignment: MainAxisAlignment.spaceBetween,
                     children: <Widget>[
                       IconLabelButton(
-                        onPressed: _showEditProjectGroupDialog,
+                        onPressed: () => _showEditProjectGroupDialog(),
                         iconPadding: _buttonIconPadding,
-                        icon: Image.network(
-                          'icons/edit.svg',
-                          width: _iconBoxSide,
-                          height: _iconBoxSide,
-                          fit: BoxFit.contain,
-                          color: theme.primaryColor,
-                        ),
-                        label: CommonStrings.edit,
-                        labelStyle: TextStyle(
-                          color: theme.primaryColor,
-                        ),
+                        iconBuilder: (context, isHovered) {
+                          return Image.network(
+                            'icons/edit.svg',
+                            width: _iconBoxSide,
+                            height: _iconBoxSide,
+                            fit: BoxFit.contain,
+                            color: isHovered
+                                ? theme.primaryButtonStyle.hoverColor
+                                : theme.primaryButtonStyle.color,
```

Comment:
Would it be better to extract it to the variable to avoid duplication? The same for `theme.accentButtonStyle`.